eng/Directory.Build.Data.targets

```diff
@@ -118,7 +118,8 @@
 
   <ItemGroup Condition="'$(UseProjectReferenceToAzureClients)' == 'true'">
     <!-- Find all Azure projects -->
-    <AzureProjects Include="$(RepoRoot)/sdk/*/Azure.*/src/Azure.*.csproj" ProjectPath="%(Identity)" />
+    <!-- TODO (kasobol-msft) undo this hack with Azure.Core.Experimental -->
```

Comment:
TODOs in this file.

Reply:
This TODO (and other TODOs) will stay until BinaryData leaves Experimental space. This will be addressed later when this goes to master. For now this will stay in feature branch.